components/clp-py-utils/clp_py_utils/s3_utils.py

```diff
@@ -116,6 +119,8 @@ def generate_container_auth_options(
     ):
         output_storages_by_component_type = [clp_config.archive_output.storage]
         input_storage_needed = True
+    elif component_type in (ARCHIVE_MANAGER_PSEUDO_COMPONENT_NAME,):
```

Comment:
If we rename this constant, we should rename `component_type` to something like `container_type`.

Reply:
sure, let's go with something simple -> `container_type`